app/assets/javascripts/channels/friend.es6

```diff
@@ -0,0 +1,34 @@
+App.friend = App.cable.subscriptions.create("FriendChannel", {
+  // connected: function() {
+  // },
+
+  // disconnected: function() {
+  // },
+
+  received: function(data) {
+    switch (data.action) {
+      case "checkin":
+        if ($(".c-friends.a-show_device").length === 0) { return };
+
+        if(data.privilege === 'complete'){
+          gon.checkins.unshift(data.msg);
+        } else {
+          gon.checkins = [data.msg];
+        }
+
+        COPO.maps.refreshMarkers(gon.checkins);
+
+        break;
+
+      case "destroy":
+        if ($(".c-friends.a-show_device").length === 0) { return };
```

Comment:
Yes, jslint recommends it as apparently using '==' you can get some weird cases where things don't equal what you'd expect. Just using === everywhere to be consistent.